governator-core/src/main/java/com/netflix/governator/AbstractLifecycleFeature.java

```diff
@@ -3,52 +3,44 @@
 import java.lang.reflect.Field;
 import java.lang.reflect.Method;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Base utility class for field and method based lifecycle features.  This class
  * deals with identifying all the fields and methods for the type and its superclasses
- * and interfaces and delgating the actual processing using the template methods
+ * and interfaces and delegating the actual processing using the template methods
  * getFieldAction and getMethodAction.
  * 
  * @author elandau
  */
 public abstract class AbstractLifecycleFeature implements LifecycleFeature {
```

Comment:
This is a backwards incompatible change on a public class.  An internal search doesn't turn up any obvious usage and a google search https://www.google.com/?gws_rd=ssl#q=extends+AbstractLifecycleFeature seems to think only Governator uses this.  Should we make it package private or something instead just to reduce any chance of breakage in the future?